inc/integrations/class-coral.php

```diff
@@ -472,6 +510,10 @@ private function set_username( $id, $username, $hash ) : bool {
 
 		$post = wp_insert_post( $args );
 
+		// Delete stored data for this SSO ID/username.
+		delete_transient( "get_username_{$id}" );
+		delete_transient( "username_exists_{$username}" );
```

Comment:
These are great, though they won't account for a case where these Post values were updated by some other means. Probably not a problem at all, but sometimes it's better to do these cache clearing calls as a callback on the `update_post` or similar hook.

Reply:
Cool, I moved them to `save_post_{$this->post_type}`